imgui/core.pyx

```diff
@@ -1374,6 +1393,307 @@ def is_window_collapsed():
+        if clicked:
+            imgui.text("Lorem Ipsum")
+        imgui.end()
+
+    Args:
+        text (str): Tree node label
+        closable (bool): define if the header is closable.
+        flags: TreeNode flags. See:
+            :ref:`list of available flags <treenode-flag-options>`.
+
+    Returns:
+        tuple: a ``(opened, closable)`` two-tuple indicating if item was
+        clicked and whether the header is opened or not.
+
+    .. wraps::
+        bool CollapsingHeader(const char* label, ImGuiTreeNodeFlags flags = 0)
+
+        bool CollapsingHeader(
+            const char* label,
+            bool* p_open,
+            ImGuiTreeNodeFlags flags = 0
+        )
+    """
+    cdef cimgui.bool inout_opened = closable
+    if closable is None:
+        clicked = cimgui.CollapsingHeader(_bytes(text), NULL, flags)
+    else:
+        clicked = cimgui.CollapsingHeader(_bytes(text), &inout_opened, flags)
+    return clicked, inout_opened
+
+
+def selectable(
+    str label,
+    selected=False,
+    cimgui.ImGuiTreeNodeFlags flags=0,
+    width=0,
+    height=0
+):
+    """Selectable text. Returns 'true' if the item is pressed.
+
+    Width of 0.0 will use the available width in the parent container.
+    Height of 0.0 will use the available height in the parent container.
+
+    .. visual-example::
+        :auto_layout:
+        :height: 200
+        :width: 200
+        :click: 80 40
+
+        selected = [False, False]
+        imgui.begin("Example: selectable")
+        _, selected[0] = imgui.selectable(
+            "1. I am selectable", selected[0]
+        )
+        _, selected[1] = imgui.selectable(
+            "2. I am selectable too", selected[1]
+        )
+        imgui.text("3. I am not selectable")
+        imgui.end()
+
+    Args:
+        label (str): The label.
+        selected (bool): defines if item is selected or not.
+        flags: Selectable flags. See:
+            :ref:`list of available flags <selectable-flag-options>`.
+        width (float): button width.
+        height (float): button height.
+
+    Returns:
+        tuple: a ``(opened, selected)`` two-tuple indicating if item was
+        clicked by the user and the current state of item.
+
+    .. wraps::
+        bool Selectable(
+            const char* label,
+            bool selected = false,
+            ImGuiSelectableFlags flags = 0,
+            const ImVec2& size = ImVec2(0,0)
+        )
+
+        bool Selectable(
+            const char* label,
+            bool* selected,
+            ImGuiSelectableFlags flags = 0,
+            const ImVec2& size = ImVec2(0,0)
+        )
+    """
+    cdef cimgui.bool inout_selected = selected
+    return cimgui.Selectable(
+        _bytes(label),
+        &inout_selected,
+        flags,
+        _cast_args_ImVec2(width, height)), inout_selected
+
+
+def listbox(
+    str label,
+    int current,
+    list items,
+    int items_count,
```

Comment:
Do we need this argument? We could simply use `len(items)` instead. If user wants to limit number of displayed items he easily slice list.